packages/musap-rn-kms/package.json

```diff
@@ -10,12 +10,13 @@
     "build:clean": "tsc --build --clean && tsc --build"
   },
   "dependencies": {
-    "@sphereon/musap-react-native": "0.0.1-next.140",
+    "@sphereon/ssi-types": "0.28.1-next.39",
+    "@sphereon/musap-react-native": "0.0.1-next.142",
     "@types/uuid": "^8.3.4",
     "@veramo/core": "4.2.0",
     "@veramo/key-manager": "4.2.0",
     "@veramo/kms-local": "4.2.0",
-    "react-native-get-random-values": "1.11.0",
+    "expo-crypto": "13.0.2",
```

Comment:
No, please reread my comment. These hould be peer deps

Reply:
As discussed on Slack, externalized keyAlias (as mandatory param) removing the need for any uuid